Content.Server/Communications/CommunicationsConsoleSystem.cs

```diff
@@ -255,10 +256,17 @@ private void OnSelectAlertLevelMessage(EntityUid uid, CommunicationsConsoleCompo
                     return;
                 }
 
+                // User has an id
                 if (_idCardSystem.TryFindIdCard(mob, out var id))
                 {
                     author = $"{id.Comp.FullName} ({CultureInfo.CurrentCulture.TextInfo.ToTitleCase(id.Comp.JobTitle ?? string.Empty)})".Trim();
                 }
+
+                // User does not have an id and is a borg, so use the borg's name
+                if (HasComp<BorgChassisComponent>(mob))
+                {
+                    author = Name(mob).Trim();
+                }
```

Comment:
Does Identity handle this?

Reply:
comms console uses ids not identity, it says Unknown instead of young man etc

Reply:
borgs don't have identity